todo/src/components/TodoItem.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function TodoItem(props) {
```

Comment:
```suggestion
const TodoItem = ({clickHandler, todo}) => (
    <li onClick={() => clickHandler(todo.id)} >
      {todo.item}
    </li>
  )
```

Reply:
use some cheeky ES6 syntax. plus i dont think you need to send the whole todo on clickHandler. can simply send it what it needs. the todo id
You are also missing the style of having a line thru it or not .